.gitignore

```diff
@@ -7,6 +7,7 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+.idea/misc.xml
```

Comment:
is this supposed to have a `/`

Reply:
I was actually confused about this too, this was automatically added by github desktop so maybe it doesn't need the /?